packages/react/src/components/FormLabel/FormLabel.stories.js

```diff
@@ -8,7 +8,7 @@
 import React from 'react';
 
 import FormLabel from './FormLabel';
-import { Tooltip } from '../Tooltip/next/Tooltip';
+import { Tooltip } from '../Tooltip/Tooltip';
```

Comment:
```suggestion
import { Tooltip } from '../Tooltip/next/Tooltip';
```
this import needs to stay the same for now, the `/Tooltip` folder hasn't been updated yet so this path imports the v10 component